test/test_jit.py

```diff
@@ -9670,6 +9674,57 @@ def func(x):
     ('Sequential', (torch.nn.Sigmoid(), torch.nn.Threshold(1., 2.)), ((S,),), True),
 ]
 
+
```

Comment:
Module test should use `nn_module_tests` i think, and pass the necessary params into them with different test cases, that will avoid different module tests appear all in test_jit.py